### PR TITLE
UKPRN Nullable format validator

### PR DESCRIPTION
**Pull request by shaun-downey-education**
- Added an additional nullable Ukrpn validator as a placeholder to replace the existing once changed to non nullable.

### CODE REVIEW

File: src/SFA.DAS.PR.Application/Common/Validators/UkprnFormatValidator.cs [hunk 1]
```diff
@@ -13,5 +13,12 @@ public static class UkprnFormatValidator
                                .WithMessage(UkprnFormatValidationMessage)
                                .When(model => model.Ukprn.HasValue);
         }
+
+        public static IRuleBuilderOptions<T, long?> CheckNullableUkprnFormat<T>(this IRuleBuilder<T, long?> ruleBuilder) where T : INullableUkprnEntity
```
Comment (DevenKShah) [inline src/SFA.DAS.PR.Application/Common/Validators/UkprnFormatValidator.cs:17]: I think this will invalidate in case of null, no? We need a test added for null value resulting in valid request.
